src/BuildingBlocks/EventBus/EventBusServiceBus/EventBusServiceBus.cs

```diff
@@ -129,10 +129,13 @@ private void RegisterSubscriptionClientMessageHandler()
                 {
                     var eventName = $"{message.Label}{INTEGRATION_EVENT_SUFIX}";
                     var messageData = Encoding.UTF8.GetString(message.Body);
-                    await ProcessEvent(eventName, messageData);
+                    var processed = await ProcessEvent(eventName, messageData);
                     
                     // Complete the message so that it is not received again.
-                    await _subscriptionClient.CompleteAsync(message.SystemProperties.LockToken);
+                    if (processed)
```

Comment:
Can we use here something like
`if (await ProcessEventAsync(eventName, messageData))`

Reply:
Yep, you could.
I just created the variable to improve readability.

Reply:
It's looks ok without variable